File: 455-assign-cookies/455-assign-cookies.cpp
class Solution {
public:
    int findContentChildren(vector<int>& g, vector<int>& s) {
        
        sort(g.begin(), g.end());
        sort(s.begin() , s.end()); 
        int cnt = 0;
        int k =0;
        for(int s1:s){
            
            if( k == g.size()) break; 
            
            if (s1 >= g[k] ){
                ++cnt;
                ++k; 
            }
            
        }
        return cnt;
        
        
        
    }
};

File: 455-assign-cookies/455-assign-cookies.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
            g.sort()
            s.sort()
            k = 0 
            cnt = 0 
            for s1 in s:
                if k ==len(g):
                    break 

                if s1 >=g[k]:
                    cnt +=1 
                    k +=1 
            return cnt 
